contents/blog/vc-or-bootstrap.md

```diff
@@ -0,0 +1,101 @@
+---
+date: 2022-11-01
+title: Raising money is less stressful than bootstrapping
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["CEO diaries", "Inside PostHog"]
+author: ["james-hawkins"]
+featuredImage: ../images/blog/posthog-ceo-diary-blog.png
+featuredImageType: full
+---
+
+Since PostHog raised Venture Capital (VC), we've had far less stress, and far more fun.
+
+There are lots of myths around what it's like run a business with VC funding - today, I'm here to dispel them.
+
+## Do board meetings with VC companies involve them pressuring expansion at all costs?
+
+Not for us.
+
+This depends entirely on who joins your board, and how you've pitched to them.
+
+Our experience has been an intense focus on finding the Ideal Customer Profile, and trying to focus as much as possible on the right users. Yes, we track revenue, signups and so on, but our board sees those as the output of building a great product for a specific set of end users. 
+
+We have to prepare for the board meetings - now that we've got the process down, these usually take a few hours to do. We keep the majority of the meeting focused on one or two discussion topics rather than presenting an update. The guidance our board have given us has outweighed the cost of setting up and running these sessions every 3 months. And, frankly, if I know someone is marking my homework, it encourages me to do a better job. Sidenote: is this the real reason we're open source?!
+
+## Control
```

Comment:
Turn into a question for consistency? "Are you losing control of the company?"

Reply:
handled by andy's suggestion for new titles :)